app/app/local_settings.py.dist

```diff
@@ -135,4 +135,11 @@ AWS_SECRET_ACCESS_KEY = 'TODO'
 S3_REPORT_BUCKET = 'TODO'
 S3_REPORT_PREFIX = 'TODO'
 
+# WEB3
+WEB3_HTTP_PROVIDER = 'TODO'
```

Comment:
could this just default to mainnet.infura.io?

Reply:
@owocki -- Sure. I can go ahead and do that in the following commit

Reply:
not as of now.  in the future, we might need a key

Reply:
@owocki -- I've updated this with the `https://mainnet.infura.io` now